src/jaxsot/core/lc.py

```diff
@@ -20,6 +21,24 @@ def gen_lightcurve(W, mmap, sigma_relative):
    return lc
 
 
+def gen_dynamic_lightcurve(W, A, X, obst, sigma_relative):
+    """light curve generator
+    Args:
+        W: geometric kernel
+        A: multiband map
```

Comment:
can you specify the dimension of these matrices? e.g. N x M (N:time dimension) M: # of bands etc

Reply:
Thank you. I modified them.